src/components/Dialog/Dialog.js

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { useSelector, useDispatch } from 'react-redux';
+import RSDialog from 'react-spatial/components/Dialog';
+import { setDialogVisible } from '../../model/app/actions';
+
+import './Dialog.scss';
+
+const propTypes = {
+  name: PropTypes.string.isRequired,
+  title: PropTypes.element,
+  body: PropTypes.element,
+  footer: PropTypes.element,
+};
+
+const defaultProps = {
+  title: null,
+  body: null,
+  footer: null,
+};
+
+const hideDialog = dispatch => () => {
```

Comment:
@oterral do we really need this function to call a function with a function? :wink:

Reply:
ahah not anymore indeed. Thx for pointing this out